manager/testdata/e2e/module-implicit-copy-kafka-to-s3-stream.yaml

```diff
@@ -83,4 +83,17 @@ spec:
       name: "encrypt"
       level: 2 # column
   chart: ghcr.io/the-mesh-for-data/m4d-kafka:0.1.0
+  expected:
```

Comment:
i think this is very complicated and error prone to have to list each resource and its expected status, is there no way to cover some common status codes (as found in a Pod) and require StreanTransfer and the likes to use the same status field as Pod does? and then we could omit this definition and simply provide an API call StatusResources which would aggregate in the same way a Pod aggregated Container status